pkg/api/v1beta3/dynakube/extensions.go

```diff
@@ -50,6 +50,10 @@ type ExtensionExecutionControllerSpec struct {
 	// +kubebuilder:validation:Optional
 	CustomConfig string `json:"customConfig,omitempty"`
 
+	// Defines name of Secret containing certificates for custom extensions signature validation
+	// +kubebuilder:validation:Optional
+	CustomExtensionCertificates string `json:"customExtensionCertificates,omitempty"`
```

Comment:
Couldn't it be just `Certificates`?

1. As of right now, it reads very "stuttery", do we need `Extension` in the name, when it is already part of the `ExtensionExecution...` section?
2. If the user has to specify them they are already `Custom`, no? Example: We don't have a `CustomLabels` field, but a `Labels` field, as "set-by-the-user" already means `Custom` (IMO)
   - Sidenote: We have `CustomPullSecret` which I have the same problems with.

Would the name `Certificates` confuse customers, because the of the higher level `TrustedCAs` field? (it is kinda confusing me 😅 )
- Maybe name it for what is used for: (if it's that specific)
    - According to the doc string, it could be: `SignatureValidationCAs` 
- Or just tell the user to put it into the `TrustedCAs` ConfigMap 🤷 
    - (This is way out of scope for this PR) `TrustedCAs` is a ConfigMap, that could have multiple keys (it only has 1 right now, `certs`), wouldn't it be nicer to have different keys for different certs (an `extensions` key for this cert)?

TLDR:  I am just questioning the whole way we ask the user to specify the certificates 😅  (IMO it's confusing)

Reply:
@StefanHauth  can you please reply here?

Reply:
had discussions offline:
- The separate field is necessary, so that is fine, implementation also looks fine
- The **problem** is that the name `customExtensionCertificates` is rather confusing, we should be more specific.
   - This might stem from the fact that our current certificate-related fields(`trustedCAs` and `tlsSecretName`) are also kinda badly named  😬

Reply:
`CustomExtension` is a known and used  term by itself in the world of extensions. It stands for extensions, that customers build themselves as opposite to extensions that come ootb in Dynatrace. Those certificates are used to validate the signature of custom extensions. 
These certificates have nothing to do with validating HTTPs connections, hence I don't see them in the trusteCAs, which get propagated to all components. These "CustomExtension" certificates are very, very specific to the EEC.

`CustomExtensionSignatureValidationCertificates` would be the over-correct name for them, but that's a wee bit longish.

Reply:
that's why we have description:
```
                        description: Defines name of Secret containing certificates
                          for custom extensions signature validation
```
anyway.

are we good to merging it? I need approvals guys!
cc @StefanHauth / @0sewa0

Reply:
it make sense, thank you for the explanation! 😘

Reply:
IMHO the field name very well describes its purpose for someone who is familiar with the extensions topic.